packages/cashier/src/components/page-container/page-container.scss

```diff
@@ -22,7 +22,7 @@
         flex: 1;
         flex-direction: column;
         gap: 1.6rem;
-        max-width: 31.2rem;
+        max-width: 30.2rem;
```

Comment:
27.8rem (sidebar right) + 2.4rem (container left margin)

Reply:
Can these use variables? Either css variables of sass variables to do the maths for this?